packit/upstream.py

```diff
@@ -260,16 +267,27 @@ def create_patches(
                 "--",
                 ".",
             ] + [f":(exclude){file_to_ignore}" for file_to_ignore in files_to_ignore]
-            patch_file = run_command(
+            git_format_p_out = run_command(
                 cmd=git_diff_cmd,
                 cwd=self.local_project.working_dir,
                 output=True,
                 decode=True,
             )
 
-            if patch_file:
-                msg = f"{commit.summary}\nAuthor: {commit.author.name} <{commit.author.email}>"
-                patch_list.append((os.path.basename(patch_file.strip()), msg))
+            # there can be multiple patches in a merge request
+            # so `git format-patch` can contain multiple files in the output
+            if git_format_p_out:
+                for patch_file_str in git_format_p_out.split("\n"):
+                    if not patch_file_str:
+                        continue
+                    prefix = format(patch_counter, f"0{order}")
+                    patch_file = Path(patch_file_str)
+                    patch_name = prefix + "-" + patch_file.name
```

Comment:
```suggestion
                    patch_name = f"{prefix}-{patch_file.name}"
```

So we have only two methods for string formatting.